cisco_aci/datadog_checks/cisco_aci/cisco.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
 
     def check(self, instance):
         self.log.info("Starting Cisco Check")
-        start = datetime.datetime.now()
+        start = datetime.datetime.utcnow()
```

Comment:
In this case is a bit irrelevant because it is just used to calculate a time delta

Reply:
If you have a DST change between `start` and `end`, you might not have the right value :)

It's an edge case, but cheap to avoid.

Reply:
Might prevent the edge case when we have a DST change on local machine between `start` and `end`, we might not have the right delta.